src/solution_0ca9ddb6.py

```diff
@@ -34,10 +47,16 @@ def solve(input_grid):
     solve is the function called to transform the input values into the expected output values
     :param input_grid: a single input grid from the list of grids
     :return: output_grid: a single output grid in numpy format after processing.
+
+    >>> test_input_grid = [[0, 0, 0],[0, 2, 0],[0, 0, 0]]
+    >>> solve(test_input_grid)
+    array([[4, 0, 4],
+           [0, 2, 0],
+           [4, 0, 4]])
     """
     numpy_grid = np.array(input_grid)
-    
-    for ix, iy in np.ndindex(numpy_grid.shape):
+
+    for (ix, iy), value in np.ndenumerate(numpy_grid):
         if numpy_grid[ix, iy] == 1 or numpy_grid[ix, iy] == 2:
```

Comment:
consider changing if numpy_grid[ix, iy] == 1 or numpy_grid[ix, iy] == 2: to if value in [1, 2]: (at least use value as you have it there)

Reply:
@jsphconnell I changed the code - thought about it further and I think makes sense

Reply:
@Enaitz Looks good - a great suggestion. Tested it locally and it still works. Lets merge this in, can you approve and merge as i opened PR?